README.md

```diff
@@ -2,3 +2,4 @@
 
 [![Actions Status](https://github.com/Porodadestroyer/python-project-lvl1/workflows/hexlet-check/badge.svg)](https://github.com/Porodadestroyer/python-project-lvl1/actions)
 [![Linter status](https://github.com/Porodadestroyer/python-project-lvl1/actions/workflows/pre-commit.yml/badge.svg)](https://github.com/Porodadestroyer/python-project-lvl1/actions)
+[![Maintainability](https://api.codeclimate.com/v1/badges/a99a88d28ad37a79dbf6/maintainability)](https://codeclimate.com/github/codeclimate/codeclimate/maintainability)
```

Comment:
link should point to your project, e. g. `https://codeclimate.com/github/Porodadestroyer/python-project-lvl1/maintainability`, now it points to codeclimate repo